baitap-19-10-2020/bai-1.js

```diff
@@ -0,0 +1,17 @@
+const salaryCalculation = (salary) => {
+    if(salary >= 15000000)
+    {
+        console.log("Thuế thu nhập: ",salary * 30 / 100)
+        console.log("Lương ròng: ",salary * 70 / 100)

```

Comment:
```suggestion
        console.log('Thuế thu nhập: ', salary * 30/100)
        console.log('Lương ròng: ', salary * 70/100)
```
Nhìn kỹ cách viết dấu cách
dùng nháy `'` thay cho `"`